CHANGELOG-4.0.md

```diff
@@ -10,6 +10,7 @@
 - Added ability to set a custom template for the Flash Messenger. [#13445](https://github.com/phalcon/cphalcon/issues/13445)
 - Added `forUpdate` in the Sqlite dialect to override the method from the base dialect. [#13539](https://github.com/phalcon/cphalcon/issues/13539)
 - Added `TYPE_ENUM` in the Mysql adapter. [#11368](https://github.com/phalcon/cphalcon/issues/11368)
+- Fixed  `\Phalcon\Http\Response::setFileToSend` filename last much _ 
```

Comment:
Could you please add this in the `## Changed` section?

Reply:
OK ， so sorry